pkg/util/image.go

```diff
@@ -73,7 +73,11 @@ func GetImageStorageClassName(image *harvesterv1.VirtualMachineImage) string {
 	if image.Spec.Backend == harvesterv1.VMIBackendCDI {
 		return image.Spec.TargetStorageClassName
 	}
-	return fmt.Sprintf("longhorn-%s", image.Name)
+	prefix := "longhorn"
```

Comment:
I think we should replace `longhorn` with something else since this is not correct anymore since we support 3rd party storage.